saw-remote-api/run_rpc_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+pushd $DIR/python
+
+NUM_FAILS=0
+function run_test {
+    "$@"
+    if (( $? != 0 )); then NUM_FAILS=$(($NUM_FAILS+1)); fi
+}
+
+echo "Setting up python environment for remote server clients..."
+poetry install
+
+export SAW_SERVER=$(cabal v2-exec which saw-remote-api)
```

Comment:
I really hope that we can upgrade the version of `cabal-install` that we use to `cabal-install-3.4` soon so that we can replace this with a simple `cabal list-bin saw-remote-api`. Using `cabal v2-exec which` is probably good enough for now, although I doubt that this would work on Windows. (Then again, most things don't.)

Reply:
It indeed does not work on windows currently... see subsequent commits playing with how to address this (in this case since other parts of the CI already worry about moving the binaries around, so it's not hard to special case for when `cabal v2-exec which` fails to identify an executable).

Reply:
For this PR I ended up disabling the windows tests. We should revisit this soon. (I believe there may be one or two other parts of the CI special casing for windows as well, so perhaps addressing those all together makes sense.)

Reply:
Fair enough. Do open an issue about this so that we can revisit trying to make these tests work on Windows later.